### DISCUSSION TITLE
Colum get data from accesor in laravel

### DISCUSSION THREAD

**Discussion by xx1196** [rappasoft/laravel-livewire-tables#1802] [Q&A]
Hello i have tis public function getFirstRoleNameAttribute(): ?Role
    {
        return $this->roles()->first()?->name;
    }, how i call this data in a colum? whit this way             Column::make(__("Role"), 'role_name'),
y have SQLSTATE[42S22]: Column not found: 1054 Unknown column 'users.role_name' in 'field list' (Connection: mysql, SQL: select `users`.`id` as `id`, `users`.`name` as `name`, `users`.`lastname` as `lastname`, `users`.`document` as `document`, `users`.`email` as `email`, `users`.`role_name` as `role_name`, `users`.`created_at` as `created_at` from `users` limit 10 offset 0)

**Comment by lrljoe** [comment]
You need to use a "label" if the field doesn't exist, just make sure that any fields used to generate the attribute are set in setAdditionalSelects

[Docs Are Here](https://rappasoft.com/docs/laravel-livewire-tables/v3/columns/available-methods#content-labels)
